File: Basic of Programming/Arrays/Q1 Reverse the array/Sol.py
arr = list(map(int,input("Input Numbers Seperated by space\n").strip().split(" ")))

for i in range(0, len(arr)//2 - 1):
    temp = arr[i]
    arr[i] = arr[len(arr)-i-1]
    arr[len(arr)-i-1] = temp

print(arr)

File: Basic of Programming/Arrays/Q1 Reverse the array/Sol.java
import java.util.*;
class Sol{

    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.println("Input numbers seperated by space");
        String sin = sc.nextLine();
        String arr[] = sin.trim().split(" ");
        int intArr[] = new int[arr.length];
        int i = 0;
        for (String s : arr){
            intArr[i] = Integer.parseInt(s);
            i++;
        }

        int res[] = reverseArray(intArr);

        for (int r : res)
            System.out.print(r + " ");
    }

    private static int[] reverseArray(int[] intArr) {
        int m = intArr.length /2 ;

        for (int i = 0; i <= m ; i++){
            int temp = intArr[i];
            intArr[i] = intArr[intArr.length - i - 1];
            intArr[intArr.length - i - 1] = temp;
        }
        return intArr;
    }


}